python/tvm/relay/op/vision/yolo.py

```diff
@@ -0,0 +1,53 @@
+"""Yolo operations."""
+from . import _make
+
+def yolo_reorg(data, stride=1):
+    """Yolo reorg operation. This layer reorganize the output based on the stride value.
+    Its function is mostly shape transform.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data tensor.
+
+    stride : int
+        The stride value for reorganisation.
+
+    Returns
+    -------
+    ret : relay.Expr
+        The computed result.
+    """
+    return _make.yolo_reorg(data, stride)
+
+
+def yolo_region(data):
+    """Yolo region operation used for detection.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data tensor.
+
+    Returns
+    -------
+    ret : relay.Expr
+        The computed result.
+    """
+    return _make.yolo_region(data)
+
+
+def yolov3_yolo(data):
```

Comment:
This is quite confusing as it is unclear what specifically the operation this, please write out more details about what is the operation about